.github/workflows/build-and-publish.yml

```diff
@@ -9,13 +9,13 @@ on:
 
 jobs:
   build-on-ubuntu:
-    uses: livMatS/dtool-lookup-gui/.github/workflows/build-on-ubuntu.yml@master
+    uses: livMatS/dtool-lookup-gui/.github/workflows/build-on-ubuntu.yml@2022-02-28-reorganize-build-workflows-a-bit
```

Comment:
Is this still correct once merged into master?

Reply:
No, but I have to merge that first and change back to master in a subsequent pull request, otherwise the workflows would fail in this PR...